app/assets/stylesheets/application.css.scss

```diff
@@ -40,6 +40,27 @@ legend {
   };
 }
 
+.reminder-container {
+  text-align: center;
+  margin: auto;
+  width: 80%;
+  border: 2px $dirty-white solid;
+  border-radius: 50px;
+  background-color: rgba(0,0,0,0.15);
+
+  h2 {
+    text-shadow: 0px 3px 0px #ffffff,
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0px` should be written without units as `0`